docs/milestone-2-steps.md

```diff
@@ -0,0 +1,183 @@
+- Buckets and objects not created via VMware OSE do not appear in vCloud Director: [S3C-4199](https://scality.atlassian.net/browse/S3C-4199)
+
+## Prerequisite
+
+### Prepare S3 Connector
+
+#### Install RING v8.4 + S3Connector
+
+- Documentation: <https://documentation.scality.com/RING/8.4.0/index.html>
+- Package: <https://packages.scality.com/stable_ori_8403>
+
+- [Alternate] Use install CI to install RING and S3 Connector: <https://scality.atlassian.net/wiki/spaces/OS/pages/1528496133/How-to+deploy+RING+S3Connector+on+scality.cloud+using+installci>
+
+- Upgrade S3C using the latest package generated via [Federation CI](https://eve.devsca.com/github/scality/federation/) (v7.10 is not GA)
+  Package used for dev testing: <https://eve.devsca.com/github/scality/federation/artifacts/builds/github:scality:federation:staging-7.10.0.0.r210614080650.b2c4b9c.pre-merge.00012275/s3-offline-centos7-7.10.0.0.tar.gz>
+- Documentation: <https://documentation.scality.com/S3C/7.9.0/installation/upgrading_s3c/Upgrading_from_7.4.X_to_7.4.X.html>
+
+Note: For credentials to download packages, please contact it@scality.com
+
+### Prepare network
+
+- Open ports `8600` and `8500` between stateful node and stateless node of S3 Connector (OSIS will be installed on the stateless node)
+- Add stateless node IP in Vault config under s3 interface, to allow AssumeRoleBackbeat to be called over port: `8500`. By default, localhost is allowed. Hence if the stateful and stateless nodes are the same, no config change is needed.
+
+```sh
+# Get the location of the config for vault container on the storage node you want OSIS to communicate with
+docker inspect scality-vault-1 | grep Source | grep conf
+                "Source": "/scality/g1ssd1/s3/scality-vault-1/conf",
+
+# In this case the config is located at /scality/g1ssd1/s3/scality-vault-1/conf/config.json
+# Edit the config.json by adding the IP address of stateless node in the below array.
+# Below we added `10.100.5.210`
+
+.
+.
+.
+ "interfaces": {
+        "S3": {
+            "address": "127.0.0.1",
+            "port": 8500,
+            "allowFrom": [
+                "127.0.0.1",
+                "::1",
+                "10.100.5.203",
+                "10.100.5.210"            ]
+        },
+.
+.
+.
+
+# Save the config and restart vault container
+docker restart scality-vault-1
+
+# Now stateless node on which OSIS is deployed should be able to communicate with Vault over port 8500 for AssumeRoleBackbeat calls
+```
+
+## Run OSIS
+
+### Prepare OSIS machine
+
+- Use a stateless node of S3 Connector to run the OSIS docker container. If stateless and stateful nodes are on the same storage nodes in the lap, you can deploy it there.
+
+- Login to docker registry:
+
+```sh
+# You will need access to `registry.scality.com/vmware-ose-scality-dev`. For credentials and access to the image repository please contact it@scality.com.
+docker login -u <username> registry.scality.com
```

Comment:
I think our image is open source as our code is. May be we don't need docker login at all.

Reply:
Yes, you are right, removed here: [`bc337ec` (#52)](https://github.com/scality/vmware-ose-scality/pull/52/commits/bc337ec87d8758d0be6cdf87ef99dd6f005236dc)